args/args_test.go

```diff
@@ -74,8 +75,11 @@ func TestSetupArgsCommandLine(t *testing.T) {
 		Verbose: true, Pretty: true, Hostname: "otherhost", Port: 1234,
 		Username: "dbuser", Password: "dbpwd", Config: *cfg, List: *list,
 	}
+
+	// default is overridden
+	args.Hostname = "otherhost"
```

Comment:
Also why here we need to set it up again if we set up in https://github.com/newrelic/infra-integrations-sdk/pull/94/files#diff-f7195c379a7e8237e2476c45d89123f5R60?